Tests/NSProcessInfoTests+SystemInfo.m

```diff
@@ -0,0 +1,38 @@
+#import <XCTest/XCTest.h>
+#import "NSProcessInfo+SystemInfo.h"
+#include <sys/types.h>
+#include <sys/sysctl.h>
+#include <mach/machine.h>
+
+
+@interface NSProcessInfoTests : XCTestCase
+
+@end
+
+@implementation NSProcessInfoTests
+
+- (void)testProcessorType {
+
+    // CPUs sourced from machine.h
+    // https://opensource.apple.com/source/xnu/xnu-792/osfmk/mach/machine.h.auto.html
+    XCTAssertEqualObjects([NSProcessInfo mme_stringForProcessorType:CPU_TYPE_X86 subtype:CPU_SUBTYPE_X86_ALL], @"x86_64");
+    XCTAssertEqualObjects([NSProcessInfo mme_stringForProcessorType:CPU_TYPE_X86 subtype:CPU_SUBTYPE_X86_64_H], @"x86_64");
+    XCTAssertEqualObjects([NSProcessInfo mme_stringForProcessorType:CPU_TYPE_X86_64 subtype:CPU_SUBTYPE_X86_ALL], @"x86_64");
+
+    // ARM CPUs
+    XCTAssertEqualObjects([NSProcessInfo mme_stringForProcessorType:CPU_TYPE_ARM subtype:CPU_SUBTYPE_ARM_ALL], @"arm");
```

Comment:
Opted for lowercase descriptors to match our projects supported architecture types:
` [ armv7, armv7s, arm64, arm64e ]`

This felt a bit easier to read and differentiate values than
`` [ ARMv7, ARMv7S, ARM64, ARM64E ]`

Reply:
looks good!